### DISCUSSION TITLE
Aggiunta note didattiche e o generiche

### DISCUSSION THREAD

**Discussion by bryfix25** [iisgiua/giuaschool#656] [Idee e proposte]
Buonasera, il registro di giua è molto funzionale e a parere mio ordinato, consiglio però di aggiungere insieme alla nota disciplinare, una nuova nota “didattica” così da non mettere note disciplinari anche quando non è il caso, per esempio dimenticanza libri, o il mancato svolgimento dei compiti per casa..

**Comment by trinko** [comment]
Esiste già e hai due modi diversi per inserirla.
Il primo modo sono le OSSERVAZIONI, leggibili solo dai genitori, che corrispondono esattamente alle "note didattiche" di cui parlavi
Più in generale, si possono usare le ANNOTAZIONI, che si può decidere se notificarle ai genitori (come prima), oppure se lasciarle visibili solo ai docenti, in caso di eventi che si vuole solo registrare formalmente.

**Reply by martinapavonidoc1** [reply]
però non c’è un punto per vedere tutte le annotazioni messe

**Comment by trinko** [comment]
> però non c’è un punto per vedere tutte le annotazioni messe

Vero. Lo segno come richiesta.

**Reply by Davide031** [reply]
Oppure si potrebbe aggiungere l inserimento multiplo Senza fare alunno per alunno

**Comment by ghost** [comment]
@bryfix25 Buongiorno! Avrei bisogno di parlarti, in merito a questa richiesta. Posso parlarti in privato? Mi scrivi alla mia mail: padovesi.roberto@gmail.com? Grazie!
